_features/fresh.html

```diff
@@ -2,6 +2,14 @@
 title: Fresh
 slug: fresh
 description: Always up-to-date.
+usecases:
+- As a web application developer, I want to deliver updates seamlessly, without requiring
+  user intervention and approval.
+- As a user, I don't want to wait for the new version to be installed; I always
+  want the latest version available.
 technologies:
 ---
-<p>Progressive Web Apps can work even on lack of network or unreliable connectivity. No more blank connectivity error pages nor dinosaurs running through the desert. A clear separation between UI and content in addition to offline caches and service workers allow you to store the application layout for future use.</p>
+<p>When users visit your web app, they are not prompted with a request to update,
+they simply see yout brand new UI with all your awesome functionality. As web apps, progressive web apps
+behave the same way out of the box but provide you with the means to fully customize the
+life cycle of your app.</p>
```

Comment:
> Erm, does this mean that you no longer want to reference the ability to customize the application lifecycle? That ability actually seems useful to me, especially since it's the way in which progressive web apps are different from regular web apps.

I just took another look at the home page for this site, which says, "Progressive Web Applications are like good old web sites but better. In a modern browser they exhibit super–powers and become…"

So it seems like you're trying to contrast "good old web sites" (GOWS?) with progressive web apps (PWAs) and show how the latter are better than the former. In which case, *fresh* doesn't seem like an accurate differentiator, because both GOWS and PWAs are fresh, even if they use different technologies to maintain freshness.

It might be worth talking about the benefits of being able to customize the application lifecycle. It might also be worth contrasting Service Workers with traditional ways of maintaining freshness, and explaining how Service Workers are designed to improve the reliability of application updates (perhaps by having the [ACID properties](https://en.wikipedia.org/wiki/ACID), unsure). But *freshness* itself isn't a selling point of PWAs specifically.